Software serial -> Master Branch

NielsvDam opened the pull request.

Gurvink committed:
Duw code

Gurvink committed:
robot

Commit:
Added Y axis motor driver. Instructions in file, mainly 0,1: Basic control, p: Push, don't use it. d: Waits 250 ms. 3: Calibrate, goes down slowly.

Commit:
Function changes etc.

Gurvink committed:
heen en weer

Commit:
Software Serial Implementation test 1.

Gurvink committed:
beweging

Gurvink committed:
Merge branch 'Robot-duw' of https://github.com/svenar-nl/KBS_Robotica into Robot-duw

Commit:
LegoMotorY Software Serial Master

Commit:
Copied X axis code from Robot-Duw

Commit:
Software Serial Connection completed.

Commit:
Fixed X axis control.

Commit:
Small changes, y controller overhaul.

TonioGrijpma committed:
😎😎😎😎😎😎😎🗡😢😢🍡⚡✡🧏‍♀️🧏‍♀️ oops, iets te veel. code ver

Commit:
Y sensor added.

TonioGrijpma committed:
deze koulo troep werk eindelijk

TonioGrijpma committed:
laatste stuff

NielsvDam committed:
Delete compiler.xml

NielsvDam committed:
Delete GUIThemes.java

svenar-nl commented:
Hoewel de arduino code gemerged kan worden, heeft Java daar geen mogelijkheid voor.

Zorg dat alle Java code niet veranderd wordt @AcceptableCasualties

NielsvDam committed:
Compat changes.

Honestly this shouldn't completely ruin the branches but I would not be suprised.

svenar-nl commented:
Alle Java code weggooien is ook niet de bedoeling ;)

Commit:
Java from master branch insert.

AcceptableCasualties commented:
@svenar-nl 
Heb de Java folder van de masterbranch erin gegooid.

AcceptableCasualties commented:
Handmatig Arduino folder in master branch gooien moment?
